### DISCUSSION TITLE
Add an option to make AudioEffectPitchShift slow down audio

### DISCUSSION THREAD

**Discussion by TheRealNoobytube** [godotengine/godot-proposals#8629] [Engine Core]
In the game I'm making, I want the game to slow down for a second once all the enemies in a room are killed. This is done easily with Engine.time_scale, however thats only in the visual side of things. I also want the music and sound effects to be slowed down. Currently this is only really possible by changing the pitch_scale property of an audiostreamplayer, which is extremely tedious when theres a ton of different sounds playing at once which all need to be slowed down temporarily. The pitchshift effect for busses seems like a good idea at first, but it only changes pitch independently from tempo.

This could all be done much simpler and more efficiently if there were an option for the pitchshift effect to slow down audio or if there was a separate effect for slowing down audio, though personally in my use case I'd prefer an effect that slowed down audio and lowered pitch simultaneously.

**Comment by Calinou** [comment]
This already exists as `AudioServer.playback_speed_scale`: https://github.com/godotengine/godot/blob/a9f444bbc307004c022e85e1b31fcfbd2e73566d/doc/classes/AudioServer.xml#L313-L315

Note that it affects everything though.
